lib/ansible/module_utils/facts/virtual/freebsd.py

```diff
@@ -68,6 +68,18 @@ def get_virtual_facts(self):
         if virtual_facts['virtualization_type'] == '':
             virtual_facts.update(virtual_vendor_facts)
 
+        # The machine is a hypervisor checking
+        # if vmm.ko kernel module is loaded
+        kldstat_bin = self.module.get_bin_path('kldstat')
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example.

Reply:
https://github.com/ansible/ansible/pull/85767/commits/7b86ac5cef97167409948db7c1ea6f836037636f